e2e/Conduit.postman_collection.json

```diff
@@ -49,7 +49,7 @@
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "{\"user\":{\"email\":\"{{EMAIL}}\", \"password\":\"{{PASSWORD}}\", \"username\":\"{{USERNAME}}\"}}"
+							"raw": "{\"email\":\"{{EMAIL}}\", \"password\":\"{{PASSWORD}}\", \"username\":\"{{USERNAME}}\"}"
```

Comment:
please revert those changes, this needs to stay the same, its official specs of the https://github.com/gothinkster/realworld project (https://github.com/gothinkster/realworld/blob/main/api/Conduit.postman_collection.json)